### PR TITLE
fix some incorrect symbols before executing command

### PR DESCRIPTION
**Pull request by lixiaobing1**
"$$" should be "$".

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐸
/cc @thaJeztah @SvenDowideit

**Comment by AkihiroSuda** [comment]
I thought these were intentionally added for the shell inside the container

**Comment by thaJeztah** [comment]
> I thought these were intentionally added for the shell inside the container

Looking at those examples, I think @AkihiroSuda is right. However, I do think it's slightly confusing; perhaps we should use the actual output, i.e. something like

```bash
root@f38c87f2a42d:/#
```

@vdemeester @SvenDowideit wdyt?

**Comment by vdemeester** [comment]
@thaJeztah yeah that would be better 👼

**Comment by SvenDowideit** [comment]
@lixiaobing10051267 I agree with @thaJeztah

**Comment by lixiaobing10051267** [comment]
ok, has been modified:)

File: docs/security/https/README.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ At this point, it has to be a manual thing, and I've been running it in boot2doc
 My process is as following:
 
     $ boot2docker ssh
-    $$ git clone https://github.com/docker/docker
-    $$ cd docker/docs/articles/https
-    $$ make cert
+    root@f38c87f2a42d:/# git clone https://github.com/docker/docker
+    root@f38c87f2a42d:/# cd docker/docs/articles/https
+    root@f38c87f2a42d:/# make cert
```
Comment (thaJeztah) [inline docs/security/https/README.md]: oh! In this document it probably needs to be `root@boot2docker:/#` (as it mentions that the hostname is `boot2docker` below)

**Comment by lixiaobing10051267** [comment]
ok, sorry about that.

**Comment by thaJeztah** [comment]
> ok, sorry about that

No worries, I could've been clearer :smile:

LGTM

**Pull request merged**